examples/with-netlify-cms/pages/blog/post/[slug].js

```diff
@@ -24,12 +24,31 @@ const Post = ({ blogpost }) => {
   )
 }
 
-Post.getInitialProps = async ({ query }) => {
-  const { slug } = query
-  const blogpost = await import(`../../../content/blogPosts/${slug}.md`).catch(
-    () => null
-  )
-  return { blogpost }
+export async function getStaticPaths() {
+  return {
+    paths: [
+      {
+        params: {
+          slug: '2019-09-06_its_not_the_problem_you_want_to_solve_boiiiiii',
```

Comment:
These should be automatically read from the filesystem!

Reply:
😅sry first timer here, let me try fix it

Reply:
i made a new commit, please cheeck